docs/index.md

```diff
@@ -1,199 +1,125 @@
-  controllers to manage software delivery.  With ArgoCD you can get a delivery
-  pipeline up and running pretty quickly.  The challenge is maintainability when
-  complexity increases.  When using Helm charts with Kustomize overlays for
-  sophisticated distributed applications, the complexity overhead can become
-  quite a burden.  Threeport advocates using code in a software controller
-  instead of config languages in a pipeline.  This means more work up-front and
-  changes to the delivery system are a bit more involved.  However, this
-  approach improves the maintainability of complex delivery systems.
-* ArgoCD generally pulls configuration from Git repos and applies them to
-  Kubernetes clusters.  Threeport uses a relational database to store config
-  which provides more efficient access to software controllers that need to both
-  read and write configuration details.
-
-ArgoCD and Threeport could be used in conjunction by using Threeport to
-provision and manage Kubernetes clusters with ArgoCD.  However, similar to
-Crossplane, there are a lot of overlapping concerns between the projects.  Using
-Crossplane and ArgoCD together make far more sense than using Threeport with
-either Crossplane or ArgoCD.
-
-## Summary
-
-Fundamentally, Threeport exists to reduce engineering toil and increase resource
-consumption efficiency in delivering software to its users.  This leads to
-greater development velocity as well as lowered engineering and infrastructure
-costs.
-
-It is designed and built upon the following principles:
-
-* General purpose programming languages like Go are superior to DSLs and
-  templates for defining the behavior of complex systems.  Threeport makes it as
-  easy as possible to extend it with custom controllers.
-* Use progressive disclosure in the abstractions available to users.  Dead
-  simple use cases should be trivial to configure and execute.  However, complex
-  use cases should be supported by allowing users greater level of
-  configurability in the underlying systems when needed.
-* Git repos are not great for storing the configuration of complex systems.  For
-  a system that is driven by software controllers, a database is more efficient
-  for both reads and writes by those controllers.
-
-If you'd like to try it out, visit our [getting started
-guide](guides/getting-started/).
-
-If you'd like to learn about the architecture, check out our [architecture
-overview](architecture/overview/).
+Threeport is an Application Orchestrator.
+
+App Orchestration is the next evolution for software delivery.  For years we have used
+continuous delivery pipelines and GitOps as methods for delivering software.  As
+cloud native has become more ubiquitous, these methods have become increasingly
+insufficient.
+
+Threeport offers a self-service experience to developers.  The developer's
+software delivery is facilitated by DevOps and platform engineering teams, but
```

Comment:
We use both "DevOps" and "Platform Engineering" in different ways, but I don't think we define how exactly we see the two terms relating to one another. Based on what we have written, it sounds like platform engineering would be those writing code to support software delivery, and devops is more of an admin role that defines object definitions for use by platform & development teams. Does that sound right?

Reply:
Yep.  That's right.  Delving into terms like this seems like a bit of a sidetrack in the Threeport docs.  I think that kind of content would be best served in a blog post (and these topics are planned).  When those posts are up, we'll add links to the docs to reference that stuff.